_posts/guides/2021-12-21-guide_gamemode-info-conc.md

```diff
@@ -17,22 +17,19 @@ toc_sticky: true
 
 ## What is Conc?
 
-Conc, short for "**Conc**ussion Grenade Jumping", is a classic game mode from the classic Team Fortress, involving priming a concussion grenade to explode at just the right time to propel the player throughout the map. 
-
-Momentum Mod's implementation of Conc comes from a mix of [Fortress Forever's implementation](https://github.com/fortressforever/fortressforever) combined with reverse engineering specific values from Team Fortress Classic.
+Concussion Grenades (Concs) are a weapon from Team Fortress which do not cause damage, but instead moves players away from the explosion. Conc mode features maps which can be completed though using these concs. Momentum Mod's implementation of Concs comes from a mix of [Fortress Forever's implementation](https://github.com/fortressforever/fortressforever) combined with reverse engineering specific values from Team Fortress Classic.
```

Comment:
> Concussion Grenades (Concs) are a weapon from Team Fortress
they come from team fortress classic

Reply:
conc nades were in the original release of quake team fortress, even from 1.0. 
https://www.moddb.com/mods/team-fortress/downloads/team-fortress-v10
/*
	TeamFortress 1.0	-	10/8/96

	Robin Walker, John Cook, Ian Caughley.

	Functions specific to the TeamFortress QuakeC patch.
*/

//=========================================================================
// Concussion grenade explosion function
void() ConcussionGrenadeExplode =
{
	T_RadiusBounce (self, self.owner, 240, world);

	WriteByte (MSG_BROADCAST, SVC_TEMPENTITY);
	WriteByte (MSG_BROADCAST, TE_EXPLOSION);
	WriteCoord (MSG_BROADCAST, self.origin_x);
	WriteCoord (MSG_BROADCAST, self.origin_y);
	WriteCoord (MSG_BROADCAST, self.origin_z);

	BecomeExplosion ();
};

Reply:
You should instead do "Quake Team Fortress" instead since most people know "Team Fortress" as tf2 or tfc

Reply:
i would suggest something like "The Team Fortress mod for Quake" (if it was actually a mod, I never can remember). at the very least putting quake in there somewhere so people get a general idea, as most people will see "team fortress" and think of either tfc or tf2.

Reply:
It is a mod yeah